tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MacroSuite.scala

```diff
@@ -9,13 +9,12 @@ class MacroSuite extends BaseDottySuite {
 
   /**
    * All examples based on dotty documentation:
-   *  https://dotty.epfl.ch/docs/reference/metaprogramming/toc.html
-   *  https://dotty.epfl.ch/docs/reference/metaprogramming/macros.html
-   *  https://dotty.epfl.ch/docs/reference/metaprogramming/tasty-reflect.html
+   * https://dotty.epfl.ch/docs/reference/metaprogramming/toc.html
+   * https://dotty.epfl.ch/docs/reference/metaprogramming/macros.html
+   * https://dotty.epfl.ch/docs/reference/metaprogramming/tasty-reflect.html
    *
-   *  '{ ... } OR 'ident - QuotedMacroExpr
-   *  '[ ... ] - QuotedMacroType
-   *  ${ ... } OR $ident - SplicedMacroExpr
+   * '{ ... } OR 'ident - QuotedMacroExpr '[ ... ] - QuotedMacroType ${ ... } OR $ident -
```

Comment:
We should make this a list so that it's not reformated

Reply:
i'd even reorder and reverse:
```
 - QuotedMacroType: '[ ... ]
 - QuotedMacroExpr: '{ ... } OR 'ident
 - SplicedMacroExpr: ${ ... } OR $ident
```